CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ also includes support for offline operations.
 
 *Changes merged to `main`, but not yet released on a tag.*
 
+## 3.4.0
```

Comment:
what's the reason for 3.4.0? due to lambdaAuth change or something else as well?